test/unit/runnable.spec.js

```diff
@@ -466,8 +468,13 @@ describe('Runnable(title, fn)', function () {
       it('should allow updating the timeout', function (done) {
         var spy = sinon.spy();
         var runnable = new Runnable('foo', function (done) {
-          setTimeout(spy, 1);
-          setTimeout(spy, 100);
+          setTimeout(function () {
+            spy();
+            setTimeout(function () {
+              spy();
+              done();
+            }, 50);
+          }, 10);
```

Comment:
I tweaked this to have a bit smaller of a `setTimeout`.

Reply:
Everything else looks good so I'm happy to approve, but I'm too new to comment on this one, hoping someone else can weigh in to confirm no breaking changes or anything!